### ISSUE TITLE
[no-duplicates] regression: normal and inline-type-specifier imports are not detected

### ISSUE THREAD

**Issue by latin-1** [import-js/eslint-plugin-import#2677]
Input:

```ts
import { A } from "./mod";
import { type B } from "./mod";
```

Output:

```ts
// 2.26
import { A, type B } from "./mod";
```

```ts
// 2.27
import { A } from "./mod";
import { type B } from "./mod";
```

**Comment by ljharb** [comment]
Duplicate of #2675; there's no need for more than one issue to cover no-duplicates with inline type specifiers.
